packages/js/src/generators/library/library.spec.ts

```diff
@@ -1152,4 +1152,20 @@ describe('lib', () => {
       expect(project.targets.build.options.assets).toEqual([]);
     });
   });
+
+  describe('--simpleName', () => {
+    it('should generate a simple name', async () => {
```

Comment:
This test isn't really checking that the `simpleName` flag did anything. It should pass a `directory` and then use `readProjectConfiguration('my-lib')` to ensure that the generated name doesn't contain the directory.

Reply:
Thanks, oversight on my part I will correct that. 

I don't think I can use `readProjectConfiguration` to verify, as its the name of the file that is created that is simplified rather than the index file and it is not available in any of the options in the project configuration. I have tests covering that the file (with the simple name) exists and is exported correctly in the index file.

Let me know if there is any thing else you would like tested, thanks